frontend/src/index.tsx

```diff
@@ -7,9 +7,7 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 )
 root.render(
-  <React.StrictMode>
```

Comment:
any reason were removing strict mode?

Reply:
Oh i thought strict mode would do 2 POST request and which would send 2 emails to the user.